raiden/tests/utils/mocks.py

```diff
@@ -32,7 +33,7 @@ class MockJSONRPCClient:
     def __init__(self, address: Address):
         # To be manually set by each test
         self.balances_mapping: Dict[Address, TokenAmount] = {}
-        self.chain_id = ChainID(17)
+        self.chain_id = ChainID(UNIT_CHAIN_ID)
```

Comment:
The value of the the `chain_id` is used for the private chain and the Raiden configuration, passing the fixture value would require a lot of changes, so instead I just changed the hard coded value to match the fixture expected value.